src/gretel_trainer/relational/model_config.py

```diff
@@ -14,6 +14,12 @@ def _model_name(workflow: str, table: str) -> str:
     return f"{workflow}-{ok_table_name}"
 
 
+def make_evaluate_config(table: str) -> Dict[str, Any]:
```

Comment:
This is nice because _down the line_ if we decide to include more evaluate tasks (I've been chatting very preliminarily with Nicole about this potential), we could edit that here.